create-operator-bundle.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
+set -x
+
+#setting sudo
+sudo echo "setting sudo root"
+
+# Initialization
+DEFAULT_CONTAINER_EXECUTABLE="docker"
```

Comment:
we already have this variable "TNF_CONTAINER_CLIENT"
it's better to use the same name.

Reply:
fixed